catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/solr/impl/ValidationQueryFactoryTest.java

```diff
@@ -103,7 +103,7 @@ public void testSearchInvalidAndNotShow()
                 .text("sample"));
         assertThat(filterAdapter.adapt(query, testValidationQueryDelegate), is(true));
         QueryRequest sendQuery = new QueryRequestImpl(query);
-        QueryRequest returnQuery = validationQueryFactory.getQueryRequestWithValidationFilter(sendQuery, false);
+        QueryRequest returnQuery = validationQueryFactory.getQueryRequestWithValidationFilter(sendQuery, false, false);
```

Comment:
I think there are tests for `false`, `true`. The ones that don't pass anything specifically do the default method, which is `false`, `true`
I do need to add a test for `true`, `false` though